DigitalLearningSolutions.Web/Controllers/RegisterController.cs

```diff
@@ -153,27 +153,43 @@ public IActionResult Summary(SummaryViewModel model)
             }
 
             var baseUrl = ConfigHelper.GetAppConfig()["CurrentSystemBaseUrl"];
-            var candidateNumber = registrationService.RegisterDelegate(RegistrationMappingHelper.MapToDelegateRegistrationModel(data), baseUrl);
+            var (candidateNumber, approved) = registrationService.RegisterDelegate(RegistrationMappingHelper.MapToDelegateRegistrationModel(data), baseUrl);
             if (candidateNumber == "-1")
             {
                 return RedirectToAction("Error", "LearningSolutions");
             }
             TempData.Clear();
             TempData.Add("candidateNumber", candidateNumber);
+            TempData.Add("approved", approved);
+            TempData.Add("centreId", data.LearnerInformationViewModel.Centre);
             return RedirectToAction("Confirmation");
         }
 
         [HttpGet]
         public IActionResult Confirmation()
         {
             var candidateNumber = (string?)TempData.Peek("candidateNumber");
-            if (candidateNumber == null)
+            var approvedNullable = (bool?)TempData.Peek("approved");
+            var centreIdNullable = (int?)TempData.Peek("centreId");
+            if (candidateNumber == null || approvedNullable == null || centreIdNullable == null)
             {
                 return RedirectToAction("Index");
             }
 
-            var viewModel = new ConfirmationViewModel(candidateNumber);
-            return View(viewModel);
+            var approved = (bool)approvedNullable;
+            var centreId = (int)centreIdNullable;
+
+            if (!approved)
+            {
+                var bannerText = centresDataService.GetBannerText(centreId);

```

Comment:
Should we get the bannerText as part of the new VC for centre details (VCs can have DI, e.g. LogoViewComponent)?

Reply:
Ooh nice catch, I should have thought of that!